lib/src/protoc_plugin_download.dart

```diff
@@ -85,28 +113,21 @@ Future<File> fetchProtocPlugin(
     try {
       // If the plugin has not been downloaded yet, download it.
       if (!await versionDirectory.exists()) {
-        // Download and unzip the .zip file containing protoc and Google .proto files.
-        await unzipUri(
-          _protocPluginUriFromVersion(version),
-          versionDirectory,
-          // Only extract the protoc_plugin from the Protobuf Git repository.
-              (file) => packages.contains(path.split(file.name)[1]),
-        );
+        await fetcher.fetchInto(versionDirectory);
 
         // Fetch protoc_plugin package dependencies.
-        await Future.wait(packages.map((pkg) =>
-            ProcessExtensions.runSafely(
+        await Future.wait(packages.map((pkg) => ProcessExtensions.runSafely(
```

Comment:
I believe, it's not necessary to `pub get` the dependencies if `protoc_plugin` was supplied from the `pubspec.yaml`.